storage/testdb/testdb.go

```diff
@@ -39,21 +39,22 @@ const (
 	// instance URI to use.
 	MySQLURIEnv = "TEST_MYSQL_URI"
 
-	defaultTestMySQLURI = "root@tcp(127.0.0.1)"
+	// Note: sql.Open requires the URI to end with a slash.
+	defaultTestMySQLURI = "root@tcp(127.0.0.1)/"
 )
 
 var (
 	trillianSQL = testonly.RelativeToPackage("../mysql/schema/storage.sql")
 )
 
-// mysqlURI returns the connection URI to use for tests.
-// It will prefer the value in the MYSQL_URI env var, and if empty/unset, it'll
-// return the defaultTestMySQLURI value.
+// mysqlURI returns the MySQL connection URI to use for tests. It returns the
+// value in the TEST_MYSQL_URI env var, if not empty, otherwise falls back to
```

Comment:
How about mentioning the env var as defined by `MySQLURIEnv`? If the assignment on L40 changes, the mention of `TEST_MYSQL_URI` no longer holds.

Reply:
Excellent suggestion, done.